### PR TITLE
USWDS-Site - a11y: publish breadcrumb checklist

### PR DESCRIPTION
**Pull request by cathybaptista**
# Summary

Added accessibility tests page for the breadcrumb component to inform users of accessibility test results.

> [!Important]
> We need to update the changelog dates before merge.

## Related issue

Closes #[2854](https://github.com/uswds/uswds-site/issues/2854)

[Changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cb-publish-breadcrumb-checklist/components/breadcrumb/#latest-updates) 

## Preview link

Preview link:
[Link to breadcrumb accessibility test page
](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cb-publish-breadcrumb-checklist/components/breadcrumb/accessibility-tests/)
## Problem statement

Users need an clear and consistent way to check results of accessibility tests run on USWDS components.

## Solution

This page provides user with accessibility test results run on the breadcrumb component in a consistent format.

## Major changes

n/a

## Testing and review

Follow these steps:

- [x] Confirm that both the main component page and the accessibility tests page have the correct compliance tag at the top.
- [x] Confirm that the main component page links to the accessibility tests page in the side navigation.
- [x] Confirm that the main component page links to the accessibility tests page from the accessibility guidance section.
- [x] Confirm that the main component page shows the accessibility tests summary.
- [x] Check that accessibility tests page provides the correct counts for each test status type.
- [x] Confirm the test checklist summaries and data are accurate.
- [x] Confirm no visual issues.
- [x] Confirm there is an appropriate changelog entry on both the main component page and the accessibility tests page.
- [x] Confirm that this code follows the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
 - [x] Run git pull origin [base branch] to pull in the most recent updates from your base and check for merge conflicts. (Often, the base branch is main).
- [x] Run npm run prettier:scss to format any Sass updates.
- [x] Run npm test and confirm that all tests pass.
- [x] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.

### CODE REVIEW

**Review by finekatie — COMMENTED** [review]
I thought I saw a typo in preview but I don't see it here in the code. All good except wondering what Amy or Alex think about the wording for one gherkin.

File: _data/accessibility-tests/breadcrumb.yml
Comment (amycole501) [reply _data/accessibility-tests/breadcrumb.yml:32]: It could be reworded to say "...the breadcrumb provides clarity on your location within the site."

Comment (finekatie) [reply _data/accessibility-tests/breadcrumb.yml:32]: I like it @amycole501 , I'll make the change!

File: _data/accessibility-tests/breadcrumb.yml
Comment (mahoneycm) [inline _data/accessibility-tests/breadcrumb.yml:28]: **todo:** This test should be listed as **conditional**. The test status on the left of the spreadsheet (🔒) seems to have fallen out of sync with it's latest results column.

```suggestion
  - summary: Breadcrumb links appear in the same location and order on every page of the website.
    summary_additional: |
      When you navigate to different pages on the website, the breadcrumb links are 
      always in the same order and in the same place.
    test_status: conditional
    test_type: general
    version_tested: 3.8.2
    wcag_criterion: 3.2.3
```

File: _data/accessibility-tests/breadcrumb.yml
Comment (mahoneycm) [inline _data/accessibility-tests/breadcrumb.yml:36]: **question:** This test feels like it should be conditional. It will rely on the users to correctly list the links to add context to the website.

Thoughts @alex-hull @amycole501 ?

**Review by amycole501 — APPROVED** [review]

**Review by mahoneycm — CHANGES_REQUESTED** [review]
Looking good! I left a couple questions for a11y and request for a necessary test status change.

## Testing checklist

- [x]  Confirm that both the main component page and the accessibility tests page have the correct compliance tag at the top.
- [x]  Confirm that the main component page links to the accessibility tests page in the side navigation.
- [x]  Confirm that the main component page links to the accessibility tests page from the accessibility guidance section.
- [x]  Confirm that the main component page shows the accessibility tests summary.
- [x]  Check that accessibility tests page provides the correct counts for each test status type.
- [ ]  Confirm the test checklist summaries and data are accurate.
    - [ ]  3.2.3 consistent navigation should be **conditional**
    - [ ]  2.4.8 also conditional?
        - 2.4.8 is more of a site wide option. Breadcrumb is the solution to this
        - Still, it relies on the user implementing this correctly
    - [ ]  4.1.2 - Refers to current page listing as a link (which it currently isn’t). Also uses refers to it as “Furthers to the right” which we might want to rephrase to final list item or something along those lines.
- [x]  Confirm no visual issues.
- [x]  Confirm there is an appropriate changelog entry on both the main component page and the accessibility tests page.

File: _data/accessibility-tests/breadcrumb.yml
Comment (mahoneycm) [reply _data/accessibility-tests/breadcrumb.yml:36]: Discussed with a11y in sync, we're good to go ahead and move this to conditional 👍 

```suggestion
  - summary: Breadcrumb links provide location within a set of pages. 
    summary_additional: |
      When navigating through different pages, 
      the breadcrumb provides clarity on your location on the site.
    test_status: conditional
    test_type: general
    version_tested: 3.8.2
    wcag_criterion: 2.4.8
```

File: _data/accessibility-tests/breadcrumb.yml
Comment (mahoneycm) [reply _data/accessibility-tests/breadcrumb.yml:116]: Discussed with a11y in sync, good to move forward with this change 👍

File: _data/accessibility-tests/breadcrumb.yml
Comment (mahoneycm) [inline _data/accessibility-tests/breadcrumb.yml:75]: **todo:** Update test_status to pass

After discussion, we decided that in it's current state this component **does** pass this test.

Additional enhancement to update the `span` be wrapped in an anchor tag is being tracked in uswds/uswds#6054

```suggestion
  - summary: Keyboard can access all links in the breadcrumb. 
    summary_additional: |
      When you navigate through the breadcrumb navigation with a keyboard, 
      you can interact with every link inside it.
    github_issue_number: 6054
    test_status: pass
    test_type: keyboard
    version_tested: 3.8.2
    wcag_criterion: 2.1.1 
```

File: _data/changelogs/component-breadcrumb-accessibility.yml
Comment (mejiaj) [inline _data/changelogs/component-breadcrumb-accessibility.yml:5]: **chore:**  Will need to add date before final merge.

File: _data/changelogs/component-breadcrumb.yml
Comment (mejiaj) [inline _data/changelogs/component-breadcrumb.yml:5]: **chore:**  Will need to add date before final merge.

**Review by mejiaj — COMMENTED** [review]
LGTM pending comments being resolved.

**Review by mejiaj — APPROVED** [review]
Approving since comments have been resolved. Ready for FFR.

File: _data/changelogs/component-breadcrumb-accessibility.yml
Comment (thisisdano) [inline _data/changelogs/component-breadcrumb-accessibility.yml:5]:
```suggestion
  - date: 2024-10-16
```

File: _data/changelogs/component-breadcrumb.yml
Comment (thisisdano) [inline _data/changelogs/component-breadcrumb.yml:5]:
```suggestion
  - date: 2024-10-16
```

**Review by thisisdano — APPROVED** [review]

**Review by thisisdano — APPROVED** [review]

**Pull request merged**
